crates/yttrium/src/chain_abstraction/api/prepare.rs

```diff
@@ -211,6 +211,7 @@ impl PrepareResponseSuccess {
 #[serde(rename_all = "camelCase")]
 pub struct PrepareResponseError {
     pub error: BridgingError,
+    pub reason: Option<String>,
```

Comment:
Can we make this non-optional? Blockchain API should always return a reason. I don't think backwards compatibility is an issue.